package.json

```diff
@@ -5,26 +5,29 @@
   "main": "dist/index.js",
   "scripts": {
     "build": "ncc build src --license licenses.txt",
-    "lint": "eslint ."
+    "test": "jest --coverage"
```

Comment:
the way you added husky is not correct. please check out the docs. also, for jest please use a configuration file instead of providing cli flags and configure so that the required coverage is 100%.